server.js

```diff
@@ -11,6 +12,54 @@ var LATIN1_SYMBOLS = '¥§©ÆÖÑøøø¼';
 var Iconv  = require('iconv').Iconv;
 var iconv = new Iconv('UTF-8', 'ISO-8859-1');
 
+function parseMultipartForm(req, res, finishCb) {
+    var fields = {}, files = {};
+
+    var busboy = new Busboy({ headers: req.headers });
+    busboy.on('file', function(fieldname, file, filename, encoding, mimetype) {
+        var currentFile = {};
+        file.on('data', function(data) {
+            currentFile.name = filename;
+            currentFile.size = data.length;
+        });
+
+        file.on('end', function() {
+            files.file = currentFile;
+        });
+    });
+
+    busboy.on('field', function(fieldname, val, fieldnameTruncated, valTruncated, encoding, mimetype) {
+        fields[fieldname] = val;
+    });
+
+    busboy.on('finish', function() {
+        console.log(stringify({fields: fields, files: files}));
+
+        finishCb(req, res, {fields: fields, files: files});
+    });
+
+    busboy.on('error', function(err) {
+        console.error('error: ' + err + ': ' + JSON.stringify(err));
+        req.connection.destroy();

```

Comment:
Might it be better to let the caller handle the error, instead of closing the connection as a side effect?